src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServerWrapper.cs

```diff
@@ -78,10 +78,8 @@ public Task WaitForExitAsync()
         return _innerServer.GetRequiredService<T>();
     }
 
-    public async ValueTask DisposeAsync()
+    public void Dispose()
     {
-        await _innerServer.DisposeAsync();
```

Comment:
CLaSP is no longer going to make the `AbstractLangaugeServer` disposable (we decided it doesn't make sense since LSP uses a two-step shutdown system). Doing this to get ahead of that curve.